third_party/toolchains/tf/tf_configure.bzl

```diff
@@ -147,7 +147,7 @@ def _symlink_genrule_for_dir(
     if src_dir != None:
         src_dir = _norm_path(src_dir)
         dest_dir = _norm_path(dest_dir)
-        files = "\n".join(sorted(_read_dir(repository_ctx, src_dir).splitlines()))
+        files = "\n".join(sorted([e for e in _read_dir(repository_ctx, src_dir).splitlines() if ("/external/" not in e) and ("/absl" not in e)]))
```

Comment:
can we change it from `/absl` to `/absl/` just to be safe and consistent with the other condition?